### PR TITLE
Simplify agreement and add £2,500 figure

### PR DESCRIPTION
**Pull request by sofer**

### CODE REVIEW

**Comment by sofer** [comment]
See https://github.com/foundersandcoders/business/issues/64

**Comment by sofer** [comment]
@iteles ready to merge?

File: README.md [hunk 1]
```diff
@@ -1,70 +1,9 @@
...
-
-**If I tell FAC about a job I want to apply for, will FAC tell other graduates about the job and increase competition for it?**  
-We think the fairest thing would be to only advertise this job after a suitable period of time has passed. 
-
-**This rule makes me uneasy. I don't want to involve FAC in my conversations with prospective employers.**    
-We understand this, but as our primary source of income, this is currently a condition of joining the programme.
-
-**Are your rules compulsory for all students?**    
-We encourage current students to abide by them, but it is now a condition of accepting an offer to study at FAC.
+In addition, each place on the Founders & Coders programme costs us about £2,500 to provide. We ask you to consider either seeking employment with one of our partners or, alternatively, making a regular voluntary contribution after you graduate as a way of *'paying it forward'*, so that we can provide a place for someone in a future cohort.
```
Comment (iteles) [inline README.md:9]: Looks like you've already made the change we'd discussed here and honestly, I :heart: how you've put it. Couldn't have said it better myself.

Just the words _paying it forward_ are so indicative of our ethos, they have me like 😍

**Comment by iteles** [comment]
:tada: Is there anywhere else that we need to propagate this to? Rewording the website? Anywhere else?

**Pull request merged**